src/components/App/App.tsx

```diff
@@ -49,16 +48,11 @@ function App() {
     console.log('inside handleSubmit', selectedCountry);
     getCurrentCountry()
   }
- 
+
   useEffect(() => {
-    if (!countries.length) {
-      apiCalls.fetchCountriesData()
+    apiCalls.fetchCountriesData()
       .then((data) => setCountries(data))
-    }
-   // try using an empty array on line 59 after the closing curly. This may let us get rid of the conditional! 
```

Comment:
we gotta remove later all no need comments, but great effort to make useEffect get to work without condions